backend/Epicenter.Domain/Services/FaceService.cs

```diff
@@ -60,11 +62,19 @@ public async Task<List<RecognizedObject>> RecognizeAsync(string base64)
                     Message = "no description",
                     LastSeen = timestamp.DateTime.GetFormattedDateAndTime()
                 });
-                _timestampRepository.Add(new Timestamp()
+                if(seenBefore && timestamp.DateTime > DateTime.UtcNow.ToUTC2().AddMinutes(-1))
```

Comment:
define a constant instead of using a magic number (-1), because it's not very clear why it's here in the first place,
would be even better if you'd define it in app settings, also you could define a boolean in this method, which would be equal to this  long expression - it's all about naming